tests/models/test_models_unet_2d_condition.py

```diff
@@ -1088,3 +1089,18 @@ def test_stabilityai_sd_v2_fp16(self, seed, timestep, expected_slice):
         expected_output_slice = torch.tensor(expected_slice)
 
         assert torch_all_close(output_slice, expected_output_slice, atol=5e-3)
+
+    @require_torch_gpu
+    def test_pickle(self, seed, timestep):
+        model = self.get_unet_model(model_id="CompVis/stable-diffusion-v1-4")
+        latents = self.get_latents(seed)
+        encoder_hidden_states = self.get_encoder_hidden_states(seed)
+
+        timestep = torch.tensor([timestep], dtype=torch.long, device=torch_device)
+
+        with torch.no_grad():
+            sample = model(latents, timestep=timestep, encoder_hidden_states=encoder_hidden_states).sample
+
+        sample_copy = copy.copy(sample)
+
+        assert sample == sample_copy
```

Comment:
Can we maybe add this as a fast test only? I don't think this test actually requires a GPU validation. 

@patrickvonplaten WDYT?

Reply:
I agree, in that case we should use a smaller model maybe.